config/StrategyResolver.py

```diff
@@ -137,88 +145,153 @@ def confirm_harvest(self, before, after, tx):
             "sett.pricePerFullShare"
         )
 
-        # 80% of collected cvxCrv is deposited on helper vaults
+
+        # (100% - performanceFee) of collected cvxCrv is deposited on helper vaults
         cvx_crv_helper_vault_delta = after.balances(
             "cvxCrv", "cvxCrvHelperVault"
         ) - before.balances("cvxCrv", "cvxCrvHelperVault")
         cvx_crv_pool_delta = after.balances(
             "cvxCrv", "cvxCrvRewardsPool"
         ) - before.balances("cvxCrv", "cvxCrvRewardsPool")
```

Comment:
Shouldn't this be before - after?

Reply:
You are right, or at least I should take the absolute value of it.

Reply:
Alright, added the absolute value to the delta and harvests tests still pass:
![image](https://user-images.githubusercontent.com/25463788/152553856-6d28390e-43fc-4b64-91fb-25a8a99492d7.png)